cmd/osd/main.go

```diff
@@ -177,6 +177,11 @@ func main() {
 			Usage: "Cluster Domain Name",
 			Value: "",
 		},
+		cli.StringFlag{
+			Name:  "csidrivername",
+			Usage: "CSI Driver name",
+			Value: "osd.openstorage.org",
```

Comment:
This value should be empty so that the csi server uses the name of the driver if CSIDriverName is empty

Reply:
Fixed